cpp/velox/compute/VeloxBackend.cc

```diff
@@ -148,12 +142,14 @@ void VeloxBackend::init(const std::unordered_map<std::string, std::string>& conf
 
   // Init glog and log level.
   if (!veloxcfg->get<bool>(kDebugModeEnabled, false)) {
-    uint32_t vlogLevel = veloxcfg->get<uint32_t>(kGlogVerboseLevel, kGlogVerboseLevelDefault);
-    FLAGS_v = vlogLevel;
-    uint32_t severityLogLevel = veloxcfg->get<uint32_t>(kGlogSeverityLevel, kGlogSeverityLevelDefault);
-    FLAGS_minloglevel = severityLogLevel;
+    FLAGS_v = veloxcfg->get<uint32_t>(kGlogVerboseLevel, kGlogVerboseLevelDefault);
+    FLAGS_minloglevel = veloxcfg->get<uint32_t>(kGlogSeverityLevel, kGlogSeverityLevelDefault);
   } else {
-    FLAGS_v = 99;
+    if (veloxcfg->isValueExists(kGlogVerboseLevel)) {
+      FLAGS_v = veloxcfg->get<uint32_t>(kGlogVerboseLevel, kGlogVerboseLevelDefault);
+    } else {
+      FLAGS_v = 99;
```

Comment:
It should be better to define kGlogVerboseLevelMaximum and replace the "99" with it.

Reply:
ok.